.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,60 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: "CodeQL Security Scan"
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+  schedule:
+    - cron: '0 16 * * 4'
```

Comment:
Maybe let's do it on Sunday when the activity is low.

Reply:
And every 2nd Sunday for example ?